index.d.ts

```diff
@@ -0,0 +1,227 @@
+/// <reference types="node" />
+import { Readable, Writable, Transform } from 'stream';
+import { EventEmitter } from 'events';
+import { Agent as HttpAgent } from 'http';
+import { Agent as HttpsAgent } from 'https';
+
+import AbortController from 'abort-controller'
+import type * as fetch from 'node-fetch';
+
+declare function megajs(options: megajs.StorageOpts, cb?: megajs.errorCb): megajs.Storage;
+
+declare namespace megajs {
+
+    export function megaEncrypt(key: Buffer, options?: cryptOpts): void | Transform;
+    export function megaDecrypt(key: Buffer, options?: cryptOpts): Transform;
+    export function megaVerify(key: Buffer): void | Transform;
+
+    export class Storage extends EventEmitter {
+        api: API;
+        key: Buffer;
+        sid: string;
+        aes: AES;
+        name: string;
+        user: string;
+        email: string;
+        shareKeys: { [nodeId in string]: Buffer };
+        options: StorageOpts;
+        status: StorageStatus;
+        root: MutableFile;
+        trash: MutableFile;
+        inbox: MutableFile;
+        mounts: MutableFile[];
+        files: { [id in string]: MutableFile };
+        RSAPrivateKey: (number | number[])[];        
+        constructor(options: StorageOpts, cb?: errorCb);
+        toJSON(): StorageJSON;
+        close(cb?: noop): void;
+        static fromJSON(json: StorageJSON): Storage;
+        mkdir(opt: mkdirOpts | string, cb?: errorCb): void;
+        upload(opt: uploadOpts | string, buffer?: BufferString, cb?: uploadCb): Writable;
+        login(cb: (error: err, storage: this) => void): void;
+        getAccountInfo(cb: (error: err, account: accountInfo) => void): accountInfo;
+        reload(cb: (error: err, mount: MutableFile[]) => void): void | this;
+        reload(force: boolean, cb: (error: err, mount: MutableFile[]) => void): void | this;
```

Comment:
Just to be sure, @qgustavor Storage.mounts is just an array of `MutableFile` right? The typing `tsc` generated for mounts were making it a an array of readonly `File` array (`[Readonly<File>[]]`) which didn't make sense..

Reply:
That's right. I don't know why tsc generated that.